### DISCUSSION TITLE
add Editor Keyboard Shortcuts in editor FUXA Tutorial

### DISCUSSION THREAD

**Discussion by superkalt** [frangoteam/FUXA#398] [Ideas]
I know the whole tutorial still needs to be filled with life, but the shortcuts are very important to reduce many requests

**Comment by unocelli** [comment]
there are some [editor Shortcuts](https://github.com/frangoteam/FUXA/wiki/Tips-and-Tricks)...can you explain what you mean?

**Comment by superkalt** [comment]
![image](https://user-images.githubusercontent.com/44277174/172691574-c50368e1-7796-4f8b-9dcb-49c1efde2ebe.png)

only a icon in the top status line. so you can look it up quickly
especially important for new users

**Comment by superkalt** [comment]
![image](https://user-images.githubusercontent.com/44277174/173435916-fd50bb7a-56c9-4176-b855-1d090d3ce9f4.png)
